chunjun-connectors/chunjun-connector-jdbc-base/src/main/java/com/dtstack/chunjun/connector/jdbc/dialect/JdbcDialect.java

```diff
@@ -401,7 +452,7 @@ default KeyUtil<?, BigInteger> initKeyUtil(String incrementName, String incremen
     }
 
     /** catalog/schema/table */
-    default Function<JdbcConf, Tuple3<String, String, String>> getTableIdentify() {
-        return conf -> Tuple3.of(null, conf.getSchema(), conf.getTable());
+    default Tuple3<String, String, String> getTableIdentify(String confSchema, String confTable) {
```

Comment:
Why 'database' are not in the parameters of function 'getTableIdentity'?

Reply:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/46181516/200212202-986bbd9d-4972-4b29-b1c4-e744faca1ec6.png">
The Connection object contains only schema and table